### ISSUE TITLE
[BUG] Unable to use absolute path for User Library folder (`~/Library`) on macOS

### ISSUE THREAD

**Issue by peterkos** [oknozor/toml-bombadil#265]
**Describe the bug**
According to the quick start docs:
https://github.com/oknozor/toml-bombadil/blob/5d3495184c897f84e9651f41eb7e535da19b8537/website/content/docs/getting-started/quick-start.md?plain=1#L81-L83

I'd expect the tilde to be inferred as an absolute path (as it's a shortcut for `/Users/username`), but instead it is rendered _relativey_, from the User directory.

**To Reproduce**
1. Create a profile with the following setting:
```toml
[profiles.sublime.dots.keymap]
source = "configs/sublime/Default.sublime-keymap"
target = ["~/Library/blah"]
```
1. Run `bombadil link -p sublime`
1. Observe symlink creation error for `/Users/username/~/Library/blah`

**Expected behavior**
Above, `["~/Library/blah"]` is respected as a _root_ path, not a relative path to `$HOME`
1. Run `bombadil link -p sublime` with the above config
1. Symlink created as expected for `~/Library/blah`

**Additional context**
The current workaround is to specify Library as a relative path `target = "Library/blah"` to reference the user path, but imo it's a little confusing. Since `target` supports absolute _or_ relative paths, I find it tricky to infer:

| Path | Note |
|---|---|
| `Library` | Probably a relative path? |
| `.Library` | Definitely a relative path|
| `/Library` | Definitely a root path |
| `~/Library` | ⚠️ Should be an absolute path (but is rendered relative!) |
| `/~/Library` | ⚠️ Should be an absolute path |

tl;dr, I believe `~/Library` should be absolute, not rendered relatively :) def open to thoughts though!

**Comment by shaleh** [comment]
If ~/ is the default why are you adding ~/??
If you just HAVE to have an absolute path, why not use the real path instead of the ~? Like /home/foo or /Users/foo/ or whatever.

**Comment by peterkos** [comment]
> If ~/ is the default why are you adding ~/??

That's not always true, which is why I'm filing this bug :) 
The docs say  "`target` shall be relative to $HOME directory or absolute" -- it's ambiguous if it will be absolute or home.

(And, it's ~~macOS~~ UNIX convention to use `~` as a shortcut to `$HOME`)

(Does that make sense? I might be misreading)

**Comment by oknozor** [comment]
fixed in latest release
